build fails with poppler-0.71.0

```
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.       -I/usr/include/libpng16 -I/usr/include/poppler  -I./cupsfilters/ -march=native -mtune=native -O2 -pipe -std=c++11 -Wall  -D_GNU_SOURCE -c -o pdftoraster-pdftoraster.o `test -f 'filter/pdftoraster.cxx' || echo './'`filter/pdftoraster.cxx
filter/pdftoraster.cxx: In function ‘void outPage(PDFDoc*, Catalog*, int, SplashOutputDev*, cups_raster_t*)’:
filter/pdftoraster.cxx:1689:45: error: invalid conversion from ‘const PDFRectangle*’ to ‘PDFRectangle*’ [-fpermissive]
   PDFRectangle *mediaBox = page->getMediaBox();
                            ~~~~~~~~~~~~~~~~~^~
filter/pdftoraster.cxx:1817:6: error: ‘gTrue’ was not declared in this scope
      gTrue,gTrue,gTrue);
      ^~~~~
filter/pdftoraster.cxx:1817:6: note: suggested alternative: ‘gfree’
      gTrue,gTrue,gTrue);
      ^~~~~
      gfree
filter/pdftoraster.cxx: In function ‘int main(int, char**)’:
filter/pdftoraster.cxx:1970:20: error: invalid conversion from ‘void (*)(void*, ErrorCategory, Goffset, char*)’ {aka ‘void (*)(void*, ErrorCategory, long long int, char*)’} to ‘void (*)(void*, ErrorCategory, Goffset, const char*)’ {aka ‘
void (*)(void*, ErrorCategory, long long int, const char*)’} [-fpermissive]
   setErrorCallback(::myErrorFun,NULL);
                    ^~
In file included from /usr/include/poppler/Object.h:45,
                 from filter/pdftoraster.cxx:44:
/usr/include/poppler/Error.h:50:37: note:   initializing argument 1 of ‘void setErrorCallback(void (*)(void*, ErrorCategory, Goffset, const char*), void*)’
 extern void setErrorCallback(void (*cbk)(void *data, ErrorCategory category,
                              ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       Goffset pos, const char *msg),
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
filter/pdftoraster.cxx:2127:5: error: ‘gFalse’ was not declared in this scope
     gFalse,paperColor,gTrue
     ^~~~~~
filter/pdftoraster.cxx:2127:5: note: suggested alternative: ‘pause’
     gFalse,paperColor,gTrue
     ^~~~~~
     pause
filter/pdftoraster.cxx:2127:23: error: ‘gTrue’ was not declared in this scope
     gFalse,paperColor,gTrue
                       ^~~~~
filter/pdftoraster.cxx:2127:23: note: suggested alternative: ‘gfree’
     gFalse,paperColor,gTrue
                       ^~~~~
                       gfree
make[1]: *** [Makefile:4366: pdftoraster-pdftoraster.o] Error 1
make[1]: Leaving directory '/var/tmp/portage/net-print/cups-filters-9999/work/cups-filters-9999'
make: *** [Makefile:2076: all] Error 2
```
At least the `gTrue` and `gFalse` issues are caused by the following commit from poppler upstream:
[Replace GBool, gTrue, and gFalse by bool, true, false, resp.](https://cgit.freedesktop.org/poppler/poppler/commit/?id=163420b48bdddf9084208b3cadf04dafad52d40a)

--- a/filter/pdftoijs.cxx
+++ b/filter/pdftoijs.cxx
@@ -72,7 +72,11 @@ namespace {
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 19
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 23
 void CDECL myErrorFun(void *data, ErrorCategory category,
+#if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 70
+    Goffset pos, const char *msg)
+#else
     Goffset pos, char *msg)
+#endif /* MAJOR > 0 || MINOR >= 70 */
 #else
 void CDECL myErrorFun(void *data, ErrorCategory category,
     int pos, char *msg)

--- a/filter/pdftoopvp/pdftoopvp.cxx
+++ b/filter/pdftoopvp/pdftoopvp.cxx
@@ -114,7 +114,11 @@ static int outOnePage(PDFDoc *doc, OPVPOutputDev *opvpOut, int pg)
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 19
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 23
 void CDECL myErrorFun(void *data, ErrorCategory category,
+#if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 70
+    Goffset pos, const char *msg)
+#else
     Goffset pos, char *msg)
+#endif /* MAJOR > 0 || MINOR >= 70 */
 #else
 void CDECL myErrorFun(void *data, ErrorCategory category,
     int pos, char *msg)

--- a/filter/pdftoraster.cxx
+++ b/filter/pdftoraster.cxx
@@ -283,7 +283,11 @@ cmsHPROFILE sgray_profile()
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 19
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 23
 void CDECL myErrorFun(void *data, ErrorCategory category,
+#if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 70
+    Goffset pos, const char *msg)
+#else
     Goffset pos, char *msg)
+#endif /* MAJOR > 0 || MINOR >= 70 */
 #else
 void CDECL myErrorFun(void *data, ErrorCategory category,
     int pos, char *msg)

--- a/filter/pdftoraster.cxx
+++ b/filter/pdftoraster.cxx
@@ -1686,7 +1686,7 @@ static void outPage(PDFDoc *doc, Catalog *catalog, int pageNo,
 {
   SplashBitmap *bitmap;
   Page *page = catalog->getPage(pageNo);
-  PDFRectangle *mediaBox = page->getMediaBox();
+  PDFRectangle mediaBox = *page->getMediaBox();
   int rotate = page->getRotate();
   double paperdimensions[2], /* Physical size of the paper */
     margins[4];	/* Physical margins of print */
@@ -1696,14 +1696,14 @@ static void outPage(PDFDoc *doc, Catalog *catalog, int pageNo,
   bool landscape = 0;
 
   fprintf(stderr, "DEBUG: mediaBox = [ %f %f %f %f ]; rotate = %d\n",
-	  mediaBox->x1, mediaBox->y1, mediaBox->x2, mediaBox->y2, rotate);
-  l = mediaBox->x2 - mediaBox->x1;
+	  mediaBox.x1, mediaBox.y1, mediaBox.x2, mediaBox.y2, rotate);
+  l = mediaBox.x2 - mediaBox.x1;
   if (l < 0) l = -l;
   if (rotate == 90 || rotate == 270)
     header.PageSize[1] = (unsigned)l;
   else
     header.PageSize[0] = (unsigned)l;
-  l = mediaBox->y2 - mediaBox->y1;
+  l = mediaBox.y2 - mediaBox.y1;
   if (l < 0) l = -l;
   if (rotate == 90 || rotate == 270)
     header.PageSize[0] = (unsigned)l;

